hadoop-cloud/pom.xml

```diff
@@ -38,7 +38,32 @@
     <sbt.project.name>hadoop-cloud</sbt.project.name>
   </properties>
 
+  <build>
```

Comment:
Is this still needed after you removed the committer code?

Reply:
it's in an adjacent PR, I've just pulled in all the POM dependency changes to keep everything related to the dependency digraph in this one so it can be audited in one go.